lib/bindings/kvbm/python/kvbm/trtllm_integration/connector/kvbm_connector_leader.py

```diff
@@ -118,6 +118,12 @@ def build_connector_meta(self, scheduler_output: SchedulerOutput) -> bytes:
         output = RustSchedulerOutput()
 
         for req in scheduler_output.new_requests:
+            if not hasattr(req, "num_scheduled_tokens"):
+                raise ValueError(
+                    """num_scheduled_tokens is not found in the SchedulerOutput!
+                    This indicates you're using an older, unsupported version of TensorRT-LLM.
+                    Are you running at least TRTLLM v1.2.0rc2?"""
```

Comment:
Are you able to print out the version of trtllm which is currently being run? This would be helpful to show the user how far off they are.